src/lib/utils/migration.ts

```diff
@@ -0,0 +1,71 @@
+import { ServiceFactory, LocalStorageService } from "../classes";
+import { ChainData } from "../classes/services/chainsService";
+import { MAX_CACHED_BLOCKS } from "../constants";
+import { LocalStorageKey } from "../enums";
+
+let storageService: LocalStorageService;
+
+interface Migration {
+    version: number;
+    migrateFunction: () => void;
+}
+
+const MIGRATIONS: Migration[] = [
+    {
+        version: 1,
+        migrateFunction: migrateToVersion2,
+    },
+];
```

Comment:
Similar design to firefly:

```suggestion

type Migration = () => void;

const MIGRATIONS: Record<number, Migration> = {
    1: migrateToVersion2
};
```